src/components/ContentBook.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ContentBook = (props) => {
+  const { title, author } = props;
```

Comment:
- [ ] **[OPTIONAL]:** You could destructure the props inside the argument list, like this:

```suggestion
const ContentBook = ({ title, author }) => {
  
```
This can improve the readability of your code and avoid unnecessary extra lines. 🤓 🔧